src/main/java/by/teachmeskills/dao/impl/mysql/books/MySQLBooksDAO.java

```diff
@@ -0,0 +1,132 @@
+package by.teachmeskills.dao.impl.mysql.books;
+
+
+import by.teachmeskills.dao.books.BooksDAO;
+import by.teachmeskills.dao.exceptions.DAOException;
+import by.teachmeskills.dao.impl.mysql.MySQLAbstractDAO;
+import by.teachmeskills.module.book.BookEntity;
+import by.teachmeskills.module.book.BookFields;
+import by.teachmeskills.module.book.BookSqlQueries;
+import by.teachmeskills.module.user.UserSqlQueries;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+public class MySQLBooksDAO extends MySQLAbstractDAO<BookEntity> implements BooksDAO {
+    private MySQLBooksDAO() {
+    }
+
+    public static class MySQLBooksDAOHolder {
+        public static final MySQLBooksDAO HOLDER_INSTANCE = new MySQLBooksDAO();
+    }
+
+    public static MySQLBooksDAO getInstance() {
+        return MySQLBooksDAO.MySQLBooksDAOHolder.HOLDER_INSTANCE;
+    }
+
+    @Override
+    public BookEntity create(BookEntity entity) throws DAOException {
+        final Optional<Connection> connectionOptional = getConnection();
+        if (connectionOptional.isPresent()) {
+            try (Connection connection = connectionOptional.get();
+                 PreparedStatement preparedStatement = connection.prepareStatement(UserSqlQueries.INSERT_USER)) {
+                preparedStatement.setString(1, entity.getTitle());
+                preparedStatement.setString(2, entity.getAuthor());
+                preparedStatement.setString(3, entity.getDescription());
+                preparedStatement.setInt(4, entity.getPublishingYear());
+                preparedStatement.setDouble(5, entity.getPrice());
+                preparedStatement.executeUpdate();
+            } catch (SQLException e) {
+                LOGGER.error("Unable to get connection: " + e.getMessage());
+                throw new DAOException("Unable to create books");
+            }
+
+        }
+        return entity;
+    }
+
+    @Override
+    public BookEntity updateEntity(BookEntity entity) throws DAOException {
+        final Optional<Connection> connectionOptional = getConnection();
+        if (connectionOptional.isPresent()) {
+            try (Connection connection = connectionOptional.get();
+                 PreparedStatement statement = connection.prepareStatement(BookSqlQueries.UPDATE_BOOK)) {
```

Comment:
UPDATE_BOOK query is:
_UPDATE books SET price WHERE id = ? _
seems like here you are trying to set new price to "WHERE id = ?" expression
should be smth like
UPDATE books SET price = ? WHERE id = ? 
and pass new price and ID as parameters